modules/ROOT/pages/_partials/include-nav-steps-to-profile.adoc

```diff
@@ -1,2 +1,3 @@
 . Log in to Anypoint Platform.
-. Navigate to your user profile.
+. In the Anypoint Platform navigation bar, click the circle icon that contains your organization or business group's acronym.
```

Comment:
The circle icon contains the initials of the user, not the org or BG acronym. Am I missing anything?